scss/_settings_colors.scss

```diff
@@ -40,6 +40,7 @@ $active-background-opacity-amount: 0.15;
 // NON-SEMANTIC COLOURS
 $color-label-validated: #006b75;
 $color-code-background: rgba($color-x-dark, 0.03);
+$color-code-background-dark: rgba($color-x-light, 0.3);
```

Comment:
Is this supposed to be `0.3` or should it be `0.03` as in light version? @sowasred2012 @lyubomir-popov

Reply:
Ok, `0.03` is almost invisible

<img width="348" alt="Screenshot 2021-02-16 at 07 30 23" src="https://user-images.githubusercontent.com/83575/108026588-ee109900-7028-11eb-98dd-c1f1601edcf8.png">

So I guess 0.3 is better, we can always adjust later

<img width="347" alt="Screenshot 2021-02-16 at 07 30 31" src="https://user-images.githubusercontent.com/83575/108026617-f8329780-7028-11eb-9f28-360639d06ec6.png">